### PR TITLE
Add exspy and holospy

### PR DESCRIPTION
**Pull request by ericpre**
https://pypi.org/project/exspy
https://pypi.org/project/holospy/

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [n/a] If static libraries are linked in, the license of the static library is packaged.
- [n/a] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [n/a] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/exspy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  license_file: LICENSE
+  doc_url: https://hyperspy.org/exspy
+  dev_url: https://github.com/hyperspy/exspy
+
+extra:
+  recipe-maintainers:
+    - CSSFrancis
+    - ericpre
+    - francisco-dlp
+    - jlaehne
```
Comment (ericpre) [inline recipes/exspy/meta.yaml:60]: @CSSFrancis, @francisco-dlp, @jlaehne, can you confirm that you are willing to be maintainers?

File: recipes/holospy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  license_file: LICENSE
+  doc_url: https://hyperspy.org/holospy
+  dev_url: https://github.com/hyperspy/holospy
+
+extra:
+  recipe-maintainers:
+    - CSSFrancis
+    - ericpre
+    - francisco-dlp
+    - jlaehne
```
Comment (ericpre) [inline recipes/holospy/meta.yaml:50]: Same as above, @CSSFrancis, @francisco-dlp, @jlaehne, can you confirm that you are willing to be maintainers?

**Comment by marcelotrevisani** [comment]
waiting for the other maintainers listed to confirm they would like to be added

**Comment by CSSFrancis** [comment]
@marcelotrevisani yes I would like to be added as a maintainer!

Sorry for the delay.

**Comment by jlaehne** [comment]
> waiting for the other maintainers listed to confirm they would like to be added

@francisco-dlp

File: recipes/exspy/meta.yaml (hunk 1 above)
Comment (francisco-dlp) [reply recipes/exspy/meta.yaml:60]: Yes, I am happy to be a maintainer.

**Comment by ericpre** [comment]
@marcelotrevisani, this should be all good now, thank you!

**Comment by ericpre** [comment]
@conda-forge/help-python, ready for review!

**Comment by CSSFrancis** [comment]
@marcelotrevisani any chance we can get this merged!  Sorry for the delay on our part.  Thanks!

**Pull request merged**